sklearn/ensemble/_bagging.py

```diff
@@ -1009,14 +1090,14 @@ def predict_log_proba(self, X):
             log_proba -= np.log(self.n_estimators)
 
         else:
-            log_proba = np.log(self.predict_proba(X))
+            log_proba = np.log(self.predict_proba(X, **predict_params))
```

Comment:
the routing needs to be corrected since in `predict_log_proba` we might call `predict` or `predict_proba`. These routing alternatives also need to be tested